Gemfile.lock

```diff
@@ -11,25 +11,25 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    activesupport (6.0.3.1)
```

Comment:
Looks like I accidentally bumped the deps updating the version but nothing here looks too crazy 🤷